### PR TITLE
Changes to get github PRs synced as changelists in GUS work item using API

### PR DESCRIPTION
**Pull request by jag-j**

### CODE REVIEW

File: api/actions/createChangelist.js [hunk 1]
```diff
@@ -5,26 +5,25 @@
...
 const Issues = require('../services/Issues');
 
 module.exports = {
     eventName: GithubEvents.events.PULL_REQUEST_CLOSED,
     fn: async function(req) {
         const {
-            pull_request: { title, url, closed_at }
+            pull_request: { title, url }
         } = req.body;
         if (title.includes('@W-')) {
```
Comment (reiniergs) [inline api/actions/createChangelist.js]: I think we should use regex here to be more specific about a work item name e.g. `/@[Ww]-\d{5}/` <-- something like that

File: api/actions/createChangelist.js [hunk 2]
```diff
@@ -5,26 +5,25 @@
...
-                work: issue.sfid,
-                changelist: url,
-                checkInDate: closed_at,
-                comment: url,
-                task: title
-            });
+            const workItemName = 'W-'.concat(
+                title.split('@W-')[1].slice(0, -1)
+            );
+            const issue = await Issues.getByName(workItemName);
```
Comment (reiniergs) [inline api/actions/createChangelist.js]: what happens when the issue with that name doesn't exists ??

Comment (jag-j) [reply api/actions/createChangelist.js]: I'm now getting this directly from GUS. So, we should always get the work item Id.

File: api/actions/createChangelist.js (hunk 1 above)
Comment (jag-j) [reply api/actions/createChangelist.js]: Took care of this.

**Pull request merged**